app/static/pages/volunteer/volunteer.html

```diff
@@ -112,259 +109,155 @@ <h6 class="md-title" show-xs hide-gt-xs>
+                    oppia-foundation-mobile-and-tablet-vertical-padding-20">
+          <md-card-title class="oppia-foundation-no-padding
+                                oppia-foundation-mobile-and-tablet-horizontal-padding-20">
+            <md-card-title-text>
+              <h3 class="md-display-1
+                         volunteer-page-content-title
+                         oppia-foundation-green-text">
+                Lesson Creation
+              </h3>
+            </md-card-title-text>
+          </md-card-title>
+          <profile-carousel carousel-type="mobile"
+                            corresponding-tab-name="lesson_creation">
+          </profile-carousel>
+          <div class="oppia-foundation-mobile-and-tablet-horizontal-padding-20">
+            <ng-include src="volunteerWorkDescriptionUrls.lessonCreation"></ng-include>
+            <div class="oppia-foundation-dotted-spaced-line">
+              <br>
+            </div>
+            <ng-include src="volunteerCategoryDescriptionUrls.lessonCreation"></ng-include>
+          </div>
+        </div>
       </md-content>
     </md-card-content>
-    <profile-carousel carousel-type="mobile"
-                      corresponding-tab-name="lesson_creation">
-    </profile-carousel>
   </md-card>
-  <md-card class="volunteer-page-contact-us-form-card
-                  oppia-foundation-vertical-padding
+  <md-card class="oppia-foundation-vertical-padding
                   oppia-foundation-horizontal-padding"
+           layout="column"
            layout-fill>
     <md-card-content layout="column"
-                     layout-gt-sm="row"
+                     layout-align="center center"
                      class="volunteer-page-display-size
                             oppia-foundation-no-padding">
-      <md-content class="volunteer-page-contact-us-form-card
-                         oppia-foundation-black-text
-                         oppia-foundation-mobile-and-small-tablet-bottom-padding-40
-                         volunteer-page-description-text-box"
-                  flex-gt-sm="55">
-        <h3 class="md-display-1
-                   volunteer-page-description-headline-padding
-                   volunteer-page-content-title">
-          Get in touch
-        </h3>
-        <p>
-          Questions? Want to learn more? Wondering if your skills can help Oppia
-          grow? We&rsquo;d love to hear from you.
-        </p>
-        <p>
-          Whatever your skills, there are ways to participate in Oppia&rsquo;s
-          mission. We invite you to reach out and see how your help can make a
-          difference.
+      <md-content class="oppia-foundation-black-text
+                         oppia-foundation-white-background
+                         volunteer-page-description-text-box">
+        <p class="volunteer-page-show-interest-text">
+          If you're interested in helping out, simply let us know what you&#39;
```

Comment:
Seems like there's a space that gets rendered between the "you'" and the "re" here.

Reply:
Good catch! Should be easily fixable by moving the "re" at the beginning of line 243 to the end of this line.

Reply:
Done :)